datalad/crawler/nodes/annex.py

```diff
@@ -1326,7 +1326,7 @@ def _finalize(data):
 
             if aggregate:
                 from datalad.api import aggregate_metadata
-                aggregate_metadata(dataset=self.repo.path, guess_native_type=True)
+                aggregate_metadata(dataset='^', path=self.repo.path)
```

Comment:
I do not understand the question. All datasets in between are populated. There is no change in behavior that I am aware of. This merely adjusts the call to use this symbol instead of having to track paths -- which the crawler is not capable of.